assets/app/styles/_messages.less

```diff
@@ -160,27 +164,20 @@ ul.messenger-theme-flat .messenger-message .messenger-actions a {
 ul.messenger-theme-flat {
   .messenger-message.alert-error {
     &:before {
-      content: "\e609";
+      content: "\e61c";
       color: @brand-danger;
     }
     .messenger-message-inner:before {
-      color: #fff;
-      content: "\e608";
       background-color: transparent;
     }
   }
   .messenger-message.alert-warning {
     &:before {
-      content: "\e60c";
+      content: "\e61c";
       color: @brand-warning;
       left: 14px;
       margin-top: -11px;
     }
-    .messenger-message-inner:before {
-      color: #fff;
-      content: "\e608";
-      background-color: transparent;
-    }
   }
   .messenger-message.alert-success .messenger-message-inner:before {
     color: @brand-success;
```

Comment:
did you double check that the icons for success and info did not change data values?